Exception handling.js

```diff
@@ -1,19 +1,19 @@
-
-
-try {
-    let a = 3;
-    console.log(a);
-
-
-} catch(e) {
-    alert('let перед использованием нужно объявить')
-
-}
-
-try{
-    let b=1/0
-    console.log(b)
-
-}catch (e) {
-    alert('на ноль делить нельзя');
-}
+

```

Comment:
try to read this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Control_flow_and_error_handling